### ISSUE TITLE
Swarm.BootstrapAsync() infinite waiting When use non-exist peer information.

### ISSUE THREAD

**Issue by ipdae** [planetarium/libplanet#710]
When Swarm starting with non-exist peer information, BootstrapAsync() was infinite waiting.

**Comment by longfin** [comment]
I've tried with some test code as below

```csharp
        public async Task Test710()
        {
            var s = _swarms[0];
            var pubKey = new PublicKey(
                ByteUtil.ParseHex(
                    "02ed49dbe0f2c34d9dff8335d6dd9097f7a3ef17dfb5f048382eebc7f451a50aa1"
                )
            );
            var peer = new BoundPeer(pubKey, new DnsEndPoint("google.com", 58598), 1);
            await s.BootstrapAsync(new[] { peer }, pingSeedTimeout: 5000, findPeerTimeout: 5000);
            Assert.True(true);
        }
```

It works as expected. but when I changed `google.com` to `google`, it didn't stop properly. 

because in case `google.com`, `Swarm<T>` raises `TimeoutException` that can process properly, but if `Swarm<T>` got `google`, it fails on name resolution and got `SocketException` that can't handle properly yet. (related PR #709 )

**Comment by longfin** [comment]
It seemed to be resolved by #709
